docs/sql-data-sources-generic-options.md

```diff
@@ -119,3 +119,31 @@ To load all files recursively, you can use:
 {% include_example recursive_file_lookup r/RSparkSQLExample.R %}
 </div>
 </div>
+
+### Modification Date Filter
+
+`modifiedDateFilter` is an option used to only load files after a specified modification
```

Comment:
Shall we use `modificationTimeFilter`? the word `date` is not accurate since it can be referred to date without any hour/minute/seconds

Reply:
@gengliangwang 
This is out of date here after recent comments from @HeartSaVioR.  I updated the PR commit above and title based on his feedback.  I'm currently working on these additions with the following:

**Example Usages**
_Load all CSV files modified after date:_
`spark.read.format("csv").option("modifiedAfter","2020-06-15T05:00:00").load()`

_Load all CSV files modified before date:_
`spark.read.format("csv").option("modifiedBefore","2020-06-15T05:00:00").load()`

_Load all CSV files modified between two dates:_
`spark.read.format("csv").option("modifiedAfter","2019-01-15T05:00:00").option("modifiedBefore","2020-06-15T05:00:00").load()`

Reply:
If you agree with this approach, I can be sure any reference to date in above, docs, etc instead use time.